utils/camera.js

```diff
@@ -1,7 +1,20 @@
 'use strict';
 
+const objectAssign = require('object-assign');
```

Comment:
superweird, got an error first time I tried but now it works! will get rid of it